### PR TITLE
Test fix - BuildKit Fails on CI

### PR DESCRIPTION
**Pull request by Tanq16**
This PR is to test CI and relates to #629 .

### CODE REVIEW

**Comment by Tanq16** [comment]
I haven't seen the issue happen with this change. Since this is hard to test as the issue isn't easily reproducible, I did 2 parallel runs of CI on this PR and another ensuring that the issue happens on the other and not here.

**Comment by jonathanmetzman** [comment]
If you commit and then push to the same PR you will also do two parallel ci runs.

**Comment by Tanq16** [comment]
> If you commit and then push to the same PR you will also do two parallel ci runs.

Oh, I thought it cancels the first run. I'll do one more test this way then.

**Comment by Tanq16** [comment]
Both the CI runs passed. I'll test once to build everything to be certain that the issue doesn't occur.

**Comment by Tanq16** [comment]
Everything apart from the Eclipser and AFLplusplus_optimal pass. Those don't have the mount error. This should fix the buildkit issue.

**Comment by jonathanmetzman** [comment]
OK hopefully this fixes it. Can you get rid of the unnecessary changes you did for testing and we'll merge this?

**Comment by Tanq16** [comment]
Fixes #629 .

File: docker/benchmark-runner/Dockerfile [hunk 1]
```diff
@@ -54,13 +54,15 @@ ENV WORKDIR /out
 RUN mkdir -p $WORKDIR
 WORKDIR $WORKDIR
 
+ARG fuzzer
```
Comment (jonathanmetzman) [inline docker/benchmark-runner/Dockerfile:57]: This line is unnecessary it already is at the top of the file.

Comment (Tanq16) [reply docker/benchmark-runner/Dockerfile:57]: It gets reset after that. I couldn't get it to work without this. If I remove it, it basically returns an empty string (at least on a local run).

Comment (jonathanmetzman) [reply docker/benchmark-runner/Dockerfile:57]: Are you sure? I find this behavior extremely bizarre. Especially given that we use the same variable twice on 37 and 41. If you're sure please just leave a comment explaining why you are doing this.

Comment (Tanq16) [reply docker/benchmark-runner/Dockerfile:57]: Yes I'm pretty sure this happens (at least on a local run). I'll add the comment.

Comment (Tanq16) [reply docker/benchmark-runner/Dockerfile:57]: I'll try a CI run without it just in case.

Comment (Tanq16) [reply docker/benchmark-runner/Dockerfile:58]: Can be seen in [this link](https://github.com/google/fuzzbench/runs/959551133?check_suite_focus=true).
Made changes.

**Pull request merged**